src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -537,5 +578,25 @@ private static Set<File> toggleFiles(File folder, boolean enable) throws IOExcep
 
 		return changed;
 	}
-	
+
+	private void searchInScript(CommandSender sender, File file, String phrase, List<String> results) {
+		if (file.isDirectory()) {
+			for (File subFile : file.listFiles()) {
+				searchInScript(sender, subFile, phrase, results);
+			}
+		} else {
+			try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
```

Comment:
I don't think TWR is available in the Java version we use.

Reply:
> I don't think TWR is available in the Java version we use.

TWR was introduced in Java 7

Reply:
Wow we have wasted so much time writing `finally` blocks not realising that 😔 😔 😔